mantidproject/mantid
# Workbench use QSettings configuration

Get the QSettings to work for the workbench main window

**To test:**

1. Make sure you don't have (on linux) `~/.config/mantidproject/workbench.ini`
2. Start the workbench, move the main window to someplace else and close
3. Start the workbench and see that it remembers that location

*There is no associated issue.*

*This does not require release notes* because workbench items are not being publicized yet.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
